src/main/resources/db/migration/V20__iat_ddl.sql

```diff
@@ -0,0 +1,25 @@
+
+-- replace item_job with content_package
+DROP TABLE IF EXISTS item_job;
+
+-- content_package
+CREATE TABLE IF NOT EXISTS content_package
+(
+    id               SERIAL PRIMARY KEY,
+    item_ids         TEXT        NOT NULL,
+    format           SMALLINT    NOT NULL,
+    include_import   BOOLEAN     NOT NULL,
+    include_tutorial BOOLEAN     NOT NULL,
+    status           SMALLINT    NOT NULL,
+    exit_message     TEXT,
+    created_date     TIMESTAMPTZ NOT NULL DEFAULT current_timestamp,
+    updated_date     TIMESTAMPTZ NOT NULL DEFAULT current_timestamp
+);
```

Comment:
Small table with some auditing information.  I don't have a "created_by" VARCHAR column, since I'm not sure ITS is going to be auth-aware.  Thoughts?

Reply:
ITS will be user aware as only some people will be able to pull content.  However, we can alter this table at that time